packages/pretty-proptypes/src/HybridLayout/index.js

```diff
@@ -117,12 +129,28 @@ export default class HybridLayout extends Component<DynamicPropsProps> {
                       required
                     </HeadingRequired>
                   )}
+                  {deprecated && (
```

Comment:
Hey! Will we have to check for `deprecated` on line 103 to make sure two headings aren't rendered? 

`{required && !deprecated && defaultValue === undefined && (`

Reply:
We discussed this in the morning - it makes sense to document both pieces of information, and the chance of a required prop being deprecated is fairly low.

Reply:
> Hey! Will we have to check for `deprecated` on line 103 to make sure two headings aren't rendered?
> 
> `{required && !deprecated && defaultValue === undefined && (`

Hi Dan!! Do we want to display both "required" and "deprecated" If a prop has both (which is unlikely to happen)?

Reply:
I see, im thinking is it possible for a prop to be both deprecated and required? If so will we see a duplicated header?

Reply:
I guess if so we just display both 😂  e.g.
<img width="335" alt="Screen Shot 2022-05-12 at 4 25 36 pm" src="https://user-images.githubusercontent.com/32695390/168005745-98e391fb-15de-4c98-b235-4505a4db40f0.png">

Reply:
This is also a process thing, we won't be deprecating many required props (in these cases it would probably be a tooling powered rename - lint fixers, codemods)

Reply:
Beautiful, yeah i see where i've gone wrong now, purely looking at the logic here (aside from should/could this be allowed,) i thought that the entire heading would be duplicated but reading it again it's only the 'indicators'. Sorry about that.

Reply:
All good, I'm going to merge this! 😄